lib/src/theme/feedback_theme.dart

```diff
@@ -54,6 +55,9 @@ class FeedbackThemeData {
 
   /// Text Style of the text above of the feedback text input.
   final TextStyle bottomSheetDescriptionStyle;
+
+  /// Whether or not to expand the bottom feedback sheet when it's active.
+  final bool enableBottomSheetExpansion;
```

Comment:
Why do we need this?

Reply:
I can imagine a lot of contexts where users would not want the expanding option. For example, the default string feedback form is so small it fits without expanding. Or, maybe, a user returns a `PageView` in their `FeedbackBuilder` where each input field is a different page-in this case expansion is unnecessary and potentially confusing.